src/it/resources/expected/all_datatypes_json/cpp-headers/my_flags.hpp

```diff
@@ -9,10 +9,10 @@
 enum class MyFlags : unsigned {
     NO_FLAGS = 0,
     /** flag option comment */
-    FLAG1 = 1u << 0,
-    FLAG2 = 1u << 1,
-    FLAG3 = 1u << 2,
```

Comment:
@a4z I feel badly about this change especially in light of the change log for 1.4.0. If you can help me maintain the previous behaviour, I would be quite happy for that.

Reply:
I can give it a look, but have to warn you that the next few days are pretty busy for me so it might take a while. 
Sorry to not be able to support your velocity at the moment.